__tests__/Rulers.unit.test.ts

```diff
@@ -0,0 +1,288 @@
+import Ruler, { defaultProps, RulerProps, RulerRenderOptions } from '../src/react-ruler';
+import { createCanvas } from 'canvas';
```

Comment:
@nickbohdanov Yes, I need the package that you wrote about in this file to create the change `_canvasContext`